<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="localhost.localdomain" name="LETest" tests="4" failures="1" timestamp="12 Feb 2016 22:21:58 GMT" time="0.031" errors="0">
  <testcase name="method1" time="0.0" classname="com.test.LargestElementTest"/>
  <testcase name="method2" time="0.003" classname="com.test.LargestElementTest"/>
  <testcase name="method3" time="0.002" classname="com.test.LargestElementTest"/>
  <testcase name="method4" time="0.002" classname="com.test.LargestElementTest">
    <failure type="java.lang.ArrayIndexOutOfBoundsException" message="0">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: 0
	at com.main.LargestElement.getLargest(LargestElement.java:11)
	at com.test.LargestElementTest.method4(LargestElementTest.java:55)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- method4 -->
</testsuite> <!-- LETest -->
